### ISSUE TITLE
Only SOME tiles give a 403 Error with Google 3D Tiles

### ISSUE THREAD

**Issue by AlphaKennyTime** [vvoovv/blender-geo#389]
I'm on Blender 3.4.1 with the free version of Blosm 2.7.2

I am able to import Google 3D Tiles with "more details" for the LOD. I get a nice 3D mesh with some tiles missing. The final line in the console reads "Info: Imported 1115 3D Tiles. 519 errors occured during the import. See the System Console." Above that is a huge list of tiles that failed to process.

![blosm test](https://github.com/vvoovv/blosm/assets/123848734/ecdbba11-849c-46dd-bffd-4d585f2bcc61)

**Comment by vvoovv** [comment]
Hi @AlphaKennyTime 

Could you please show the output of the System Console?

**Comment by AlphaKennyTime** [comment]
I updated Blender to 3.6.2 and manually selected the same area again. The coordinates aren't exactly the same but they are very close. Now I only got 13 errors. This is way better now, but the errors never completely went away. 

If it matters, I had the API temporarily set to unrestricted for both tests since I couldn't figure out what restrictions I could place while still using it for Blosm. 

![way better](https://github.com/vvoovv/blosm/assets/123848734/23073b77-b4db-4338-a027-1d1314eade80)
![nice blosm](https://github.com/vvoovv/blosm/assets/123848734/5e63fa07-41d6-4cba-91db-4053fce32b62)
